sqlite_utils/db.py

```diff
@@ -1387,12 +1426,23 @@ def pks_and_rows_where(
                 row_pk = row_pk[0]
             yield row_pk, row
 
+    @property
+    def is_attached(self) -> bool:
+        return dbname(self.name) not in {'', 'main'}
+
+    @property
+    def _pragma_name(self) -> Tuple[str, str]:
+        if "." in self.name:
+            db, name = self.name.split(".")
+            return db + ".", name
+        return "", self.name
```

Comment:
this is maybe premature optimization for not changing too much of the code in an visually noisy way. It really just lets me modify all the places we do PRAGMA things without making big changes to what's there.